metadata/contributors.json

```diff
@@ -3772,6 +3772,16 @@
          "jeremymos"
       ]
    },
+   {
+      "emails" : [
+         "jesse_hertz@apple.com",
+         "jesse.hertz@apple.com"
+      ],
+      "nicks" : [
```

Comment:
We should have a GitHub username, `jhertz-apple`, right?

Reply:
fixed and updated